src/pages/Proposal/ProposalSummary/index.tsx

```diff
@@ -157,6 +161,26 @@ export const ProposalSummaryUi: React.FC<
     // no default
   }
 
+  const countdownData = useMemo(() => {
+    if (state === 'Active' && endDate) {
+      return {
+        date: endDate,
+        // DO NOT REMOVE COMMENT: needed by i18next to extract translation key
+        // t('voteProposalUi.activeUntilDate')
+        i18nKey: 'voteProposalUi.activeUntilDate',
+      };
+    }
+
+    if (state === 'Succeeded' && proposalEta && isAfter(proposalEta, new Date())) {
```

Comment:
```suggestion
    if (state === 'Queued') {
```

Reply:
the waiting period is the length of the time the proposal will be in the queued state